pyblish/util.py

```diff
@@ -12,7 +12,7 @@
 log = logging.getLogger("pyblish.util")
 
 
-def publish(context=None, plugins=None, targets=[]):
+def publish(context=None, plugins=None, targets=["default"]):
```

Comment:
Don't add it to the argument, in fact never put a list as default value for any argument; it's mutable. Meaning that if the function modifies the default value, it will persist till the next call to the function. It's a really tough quirk to debug. Better put `None` and make it a list in the function body.